data/110/873/050/9/1108730509.geojson

```diff
@@ -11,11 +11,14 @@
     "geom:longitude":27.96352,
     "iso:country":"ZA",
     "label:eng_x_preferred_longname":[
-        "Johannesburg Municipality"
+        "City of Johannesburg Metropolitan Municipality"
```

Comment:
The labels are meant to be short and sweet. This sounds like a legal name, not a display name. Should the new form go into the name space instead?

Reply:
I've updated this in a recent commit.. preferring "City of Johannesburg" as the preferred label, moving the full string to a variant.